src/metadata/libexif_handler.cc

```diff
@@ -353,8 +353,12 @@ void LibExifHandler::process_ifd(ExifContent* content, const std::shared_ptr<Cds
     }
 }
 
-void LibExifHandler::fillMetadata(std::shared_ptr<CdsItem> item)
+void LibExifHandler::fillMetadata(std::shared_ptr<CdsObject> obj)
 {
```

Comment:
If these handlers require an item then isnt it nicer to have the typing check? (as it was?)

Reply:
It's because of CdsContainer being used in some cases. I didn't want to use separate calls. Any suggestions?

Reply:
I don't have a better suggestion 😬